app/include/zmk/hid.h

```diff
@@ -10,6 +10,7 @@
 #include <zephyr/usb/class/usb_hid.h>
 
 #include <zmk/keys.h>
+#include <zmk/mouse.h>
```

Comment:
Along the same lines as my other comments, should the additions here be guarded?